daliuge-engine/dlg/apps/plasmaflight.py

```diff
@@ -0,0 +1,92 @@
+#
+#    ICRAR - International Centre for Radio Astronomy Research
+#    (c) UWA - The University of Western Australia, 2015
+#    Copyright by UWA (in the framework of the ICRAR)
+#    All rights reserved
+#
+#    This library is free software; you can redistribute it and/or
+#    modify it under the terms of the GNU Lesser General Public
+#    License as published by the Free Software Foundation; either
+#    version 2.1 of the License, or (at your option) any later version.
+#
+#    This library is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#    Lesser General Public License for more details.
+#
+#    You should have received a copy of the GNU Lesser General Public
+#    License along with this library; if not, write to the Free Software
+#    Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+#    MA 02111-1307  USA
+#
+import hashlib
+from io import BytesIO
+import logging
+from typing import Optional
+
+import pyarrow
+import pyarrow.flight as paf
+import pyarrow.plasma as plasma
+
+logger = logging.getLogger(__name__)
+
+
+class PlasmaFlightClient():
```

Comment:
This file is a copy from https://github.com/ICRAR/plasmaflight/blob/main/icrar/plasmaflight/client/plasmaflight_client.py

An alternative option of getting it here is to add icrar.plasmaflight as a package dependency for daliuge